hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -322,6 +330,37 @@ public void makeQualified(FileStatus status, URI uri, Path path,
     return new IteratorAdapter(bucket.listKeys(pathKey));
   }
 
+  @Override
+  public Map<String, List<OzoneAcl>> getAcl(String keyName) throws IOException {
```

Comment:
We can rename the api to getAcls and setAcls since we are dealing with multiple acls.